### PR TITLE
Fix wrong NativeImage method names

### LINKED ISSUE #719: NativeImage get/set pixel method names are encorrect
NativeImage uses the same pixel format as OpenCV/OpenGL. Found this whilst testing why pixel values in MineLP aren't being interpreted correctly.

To reflect this both the methods should be renamed:

NativeImage.getPixelRGBA -> NativeImage.getPixelABGR
NativeImage.setPixelRGBA -> NativeImage.setPixelABGR

### PR DESCRIPTION
**Pull request by i509VCB**
Fixes #719

### CODE REVIEW

**Comment by i509VCB** [comment]
Note the current names are incorrect since NativeImages do not use ARGB, it's just that mojang's strings specified in exceptions are wrong.

**Comment by liach** [comment]
uhh let's stick to rgba, just note this is from least to most significant bits.
It's a common practice to put values from least to most significant bits (especially when the value is array-esque), such as jvm class file versions, where major versions are in the lower bits while minor versions are in higher bits

**Comment by liach** [comment]
Also see my uv example in that issue above

**Comment by i509VCB** [comment]
> uhh let's stick to rgba

This has the same energy as using `motive` because the registry says `motive`.

I am putting my foot and and I am saying we should use the proper color format for the method name, or change to `get/setPixelColor`

**Comment by liach** [comment]
We should probably call it big/little endian rgba instead.

File: mappings/net/minecraft/client/texture/NativeImage.mapping [hunk 1]
```diff
@@ -169,6 +191,8 @@ CLASS net/minecraft/class_1011 net/minecraft/client/texture/NativeImage
 		METHOD method_4339 setPackAlignment ()V
 		METHOD method_4340 setUnpackAlignment ()V
 	CLASS class_1013 GLFormat
```
Comment (Runemoro) [inline mappings/net/minecraft/client/texture/NativeImage.mapping:196]: This is a weird name, shouldn't it be something like `ColorFormat` or `PixelFormat`?

**Review by YanisBft — APPROVED** [review]

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/client/texture/NativeImage.mapping (hunk 1 above)
Comment (i509VCB) [reply mappings/net/minecraft/client/texture/NativeImage.mapping:196]: I'll address this in a later PR

**Review by YanisBft — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**